website/api/controllers/articles/view-basic-article.js

```diff
@@ -5,11 +5,13 @@ module.exports = {
 
   description: 'Display "Blog article" page.',
 
-  urlWildcardSuffix: 'slug',
+
+  urlWildcardSuffix: 'pageUrlSuffix',
+
 
   inputs: {
-    slug : {
-      description: 'The relative path to the blog page from within this route.',
+    pageUrlSuffix : {
```

Comment:
fyi "slug" can sometimes connote that it is a string that has no slashes, at least in the context of Sails apps

Reply:
(but this has slashes)